plasmapy/formulary/shocks.py

```diff
@@ -0,0 +1,116 @@
+""" This module gathers functions relating to shocks and the properties thereof.
+"""
+__all__ = ["entropy_jump_polytropic"]
+
+
+import warnings
+from astropy import units as u
+from plasmapy.utils.exceptions import PhysicsWarning
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    check_relativistic,
+    validate_quantities,
+)
+from astropy.constants import c, a0, k_B
+from numpy import pi, exp, sqrt, log, nan, isclose
+
+
+@validate_quantities(
+    rho_1={"can_be_negative": False, "pass_equivalent_units": True},
+    rho_2={"can_be_negative": False, "pass_equivalent_units": True},
+    p_1={"can_be_negative": True, "pass_equivalent_units": True},
+    p_2={"can_be_negative": True, "pass_equivalent_units": True},
+    c_v={"can_be_negative": False},
+)
+def entropy_jump_polytropic(
+    c_v: u.J / u.K,
+    p_1: u.Pa,
+    p_2: u.Pa,
+    rho_1: u.kg / u.m ** 3,
+    rho_2: u.kg / u.m ** 3,
+    gamma=5 / 3,
+) -> u.J / u.K:
+    r"""
+    Entropy is not conserved across a shock, since a shock is
+    an irreversible, non-adiabatic process. This equation is based on the specific entropy
+    of a polytropic gas before and after the shock has passed. The shock
+    is an isolated steady, perpendicular shock, flowing through a non magnetized fluid.
+
+     .. math::
+
+        s_2 - s_1 =
+        c_v ln\left[ \frac{p_2}{p_1} \left( \frac{\rho_1}{\rho_2} \right)^{\gamma} \right]
+    Here variables indexed by 1 and 2 are upstream (pre shock)
+    and downstream (post shock) respectively.
+    This function is equivalent to Eq. 4.24 in `Drake`_.
```

Comment:
This needs rewording.  For example, there is no magnetic field in this shock formulation so there's no sense of it being perpendicular or parallel.  Additionally, the derivation uses the assumption of a planar shock.  ...I'll need to think about this more, so I'll get back to you with additional thoughts on verbiage.